src/util.js

```diff
@@ -11,6 +10,12 @@ const getRandomInteger = (a,b) => {
   return Math.floor(minInteger + Math.random()*(maxInteger - minInteger + 1));
 };
 
+const getRandomArrayElement = (array, min, max) => {
+  const randomArrayElement = array[getRandomInteger(min, max)];
+
+  return randomArrayElement;
+};
```

Comment:
Так как мы ожидаем случайный элемент из всего массива, то уже известны мин и макс. То есть можно упростить до:

```suggestion
const getRandomArrayElement = (array) => array[getRandomInteger(0, array.length)];
```

И после этого можно удалить все `*_MAX_INDEX` константы.

Reply:
Красота!)